src/components/utils/onfidoApi.js

```diff
@@ -7,6 +7,7 @@ const formatError = ({response, status}, onError) =>
   onError({status, response: JSON.parse(response)})
 
 export const uploadDocument = (data, token, onSuccess, onError) => {
+  data['sdk_validations'] = {'detect_document': 'error', 'detect_glare': 'warn'}
```

Comment:
better never to change objects in place unless it's made clear. a simple `data= {...data, sdk_validations: ...` will do the trick